relay/relay.go

```diff
@@ -133,6 +139,8 @@ type Relay struct {
 	beacon      Beacon
 	beaconState State
 
+	exp DataExporter
```

Comment:
To be fair I don't think that export is the right word for that. It's not an export - it's just another kind of storage. Persistor maybe? Or there is definitely better wording for that

Reply:
I understand, and you are right, but what is the purpose of this? The trace is for the external world; this data is internally for analyzing and debugging. Any name based on that?

Reply:
Datastore? ;) I would say DataWriter would be better